view/frontend/web/js/view/payment/method-renderer/two_payment.js

```diff
@@ -318,13 +346,41 @@ define([
                     quantity: item['qty'],
                     unit_price: parseFloat(item['price']).toFixed(2),
                     tax_amount: parseFloat(item['tax_amount']).toFixed(2),
-                    tax_rate: parseFloat(item['tax_percent']).toFixed(6),
+                    tax_rate: (parseFloat(item['tax_percent']) / 100 ).toFixed(6),
                     tax_class_name: '',
                     quantity_unit: config.orderIntentConfig.weightUnit,
                     image_url: item['thumbnail'],
                     type: item['is_virtual'] === '0' ? 'PHYSICAL' : 'DIGITAL'
                 });
             });
+
+
+            const orderIntentRequestBody = {
+                gross_amount: parseFloat(totals['grand_total']).toFixed(2),
+                invoice_type: config.orderIntentConfig.invoiceType,
+                currency: totals['base_currency_code'],
+                line_items: lineItems,
+                tax_subtotals : calculateTaxSubtotals(lineItems),
```

Comment:
think you might need to do this.calculateTaxSubtotals

Reply:
updated